src/common_subset.rs

```diff
@@ -1,4 +1,26 @@
-//! Asynchronous Common Subset algorithm.
+//! # Asynchronous Common Subset algorithm.
+//!
+//! The Asynchronous Common Subset protocol assumes a network of _N_ nodes that send signed
+//! messages to each other, with at most _f_ of them malicious, where _3 f < N_. Handling the
+//! networking and signing is the responsibility of the user: only when a message has been
+//! verified to be "from node i", it can be handed to the `CommonSubset` instance.
+//!
+//! Each participating node proposes an element for inclusion. Under the above conditions, the
+//! protocol guarantees that all of the correct nodes output the same set, consisting of at least
```

Comment:
~~participating~~ .  Not sure if needed because of dynamic HB?

...protocol guarantees that all ~~ of the~~ correct nodes ...